doc/getting_started/installation_win.rst

```diff
@@ -20,6 +20,145 @@ Windows system with the latest updates installed.
 Installing Requirements and Dependencies
 ****************************************
 
+There are 3 different ways of developing for Zephyr on Microsoft Windows.
+The first one is fully Windows native, whereas the 2 additional ones require
+emulation layers that slow down build times and are not as optimal. All of
+them are presented here for completeness, but unless you have a particular
+requirement for a UNIX tool that is not available on Windows, we strongly
+recommend you use the Windows Command Prompt for performance and minimal
+dependency set.
+
+Using Windows Command Prompt (Recommended, fastest)
+===================================================
+
+The easiest way to install the dependencies natively on Microsoft Windows is
+to use the :program:`Chocolatey` package manager (`Chocolatey website`_).
+If you prefer to install those manually then simply download the required
+packages from their respective websites.
+
+   .. note::
```

Comment:
un-indent this ``.. note::`` to the left margin (and the subsequent text to the ``n`` in ``note``)